fbw-a32nx/src/wasm/systems/a320_systems/src/hydraulic/mod.rs

```diff
@@ -3882,6 +3976,22 @@ impl SimulationElement for A320HydraulicBrakeSteerComputerUnit {
         self.autopilot_nosewheel_demand =
             Ratio::new::<ratio>(reader.read(&self.autopilot_nosewheel_demand_id));
     }
+
+    fn write(&self, writer: &mut SimulatorWriter) {
+        for (id, temp) in self
+            .brake_temperature_ids
+            .iter()
+            .zip(self.brake_temperatures)
+        {
+            // TODO: send status when not powered
```

Comment:
Should send FW with FIXME to change to "no transmission" when we get around to implementing an A429 bus simulation. The values should also be quantised (steps of 5) and limited to the appropriate range (0 to 995), as the A429 encoding doesn't currently do that for us.

Reply:
Since currently it's not simulated that the bscu is powered I added a comment.